src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const divStyle = {
+  width: '500px',
+  margin: 'auto',
+};
+
+function App() {
+  return (
+    <div style={divStyle}>
```

Comment:
- [ ] [MANDATORY] It is not best practice to use inline CSS. Read more about it in this [Article](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846).